### ISSUE TITLE
character dialog cuts off in landscape mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#39]
```
What steps will reproduce the problem?
1. Engage dialog with a character in landscape mode 
2.
3.

What is the expected output? What do you see instead?
Expected to either have all of the dialog displayed, be able to scroll down 
through the dialog, or click next to view the remainder of the dialog.

What happens is that in landscape mode only, the full dialog is not displayed, 
and clicking next does not display the missing dialog.  No scroll ability is 
available.

What version of the product are you using? On what operating system?
Android G1, latest version.


Please provide any additional information below.


```

Original issue reported on code.google.com by `practica...@gmail.com` on 29 Nov 2010 at 8:39

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. This will be fixed in 0.6.6 that should be out in 
Android Market soon. In 0.6.6, the text is scrollable.

Thanks again for reporting it.
```

Original comment by `oskar.wi...@gmail.com` on 29 Nov 2010 at 8:41
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This is now implemented. Closing this issue. Thanks again for reporting it!
```

Original comment by `oskar.wi...@gmail.com` on 29 Nov 2010 at 9:51
- Changed state: **Fixed**
